### ISSUE TITLE
Issue after downloading the new 1.10.2 release

### ISSUE THREAD

**Issue by SkullCrusher548** [ldtteam/minecolonies#927]
I downloaded the new release minecolonies-universal-1.10.2-0.7.780.jar 
[crash-2017-05-02_01.40.44-server.txt](https://github.com/Minecolonies/minecolonies/files/972637/crash-2017-05-02_01.40.44-server.txt)
and now all i do is crash and my server is telling me it cant find the signature matching the description.
![imageedit_1_9626033607](https://cloud.githubusercontent.com/assets/25522161/25650874/d81c3a1a-2fad-11e7-9b32-61fa0ebfaa72.png)

**Comment by marchermans** [comment]
Seems like we released an unsigned version. I will check this out. You say the 1.10.2-0.7.780. Let me check which branch released this.

**Comment by SkullCrusher548** [comment]
Ok ty

**Comment by marchermans** [comment]
Okey I figured it out. This was a release build that was made on one of our Linux machines. We forgot to copy over the signature that is used to sign the Jars to them, so the build system skipped a the sign step as its signature file was not available.

This is not really a problem as it is just for us, to check that when you have issues you are using one of out official files, and Minecraft Forge is just telling us that the fingerprint we told it the file would have @FINGERPRINT@ is not found on the file, as it is not signed at all.

I will make sure that @Kostronor gets the Signature and can sign the files himself.

In other words their is nothing wrong. We just forgot to copy a file. Sorry.

**Comment by SkullCrusher548** [comment]
Thank you guys!!!!

**Comment by marvin-bitterlich** [comment]
closing this
